docs/development-guide.md

```diff
@@ -183,8 +183,8 @@ Other coding conventions include:
 To push the rebuilt image, you need write access to [`gitmachete` organization on Docker Hub](https://hub.docker.com/orgs/gitmachete).
 
 ```
-docker build -t gitmachete/intellij-plugin-ci:<SEMANTIC-VERSION> .
-docker push gitmachete/intellij-plugin-ci:<SEMANTIC-VERSION>
+docker build -t gitmachete/intellij-plugin-ci:SEMANTIC-VERSION - < Dockerfile
```

Comment:
We don't do any `COPY` or `ADD` in Dockerfile, build context is completely unnecessary

Reply:
Hence also removal of .dockerignore &mdash; nothing to ignore coz the build context is empty